app/src/routes/v1/acronyms.js

```diff
@@ -6,6 +6,34 @@ const log = require('npmlog');
 const Problem = require('api-problem');
 
 const acronyms = require('../../components/acronyms');
+const permissionHelpers = require('../../components/permissionHelpers');
+
+/** fetches the acronym details */
+acronymsRouter.get('/:appAcronym', [
+], async (req, res) => {
+  // Check for required permissions. Can only fetch details for the acronyms you are associated with
+  const permissionErr = await permissionHelpers.checkAcronymPermission(req.kauth.grant.access_token.content.sub, req.params.appAcronym);
+  if (permissionErr) {
+    return res.status(403).json({
+      message: permissionErr
+    });
```

Comment:
Use the api-problem library for any non-successful responses.

Reply:
Fixed (and all below)